From f0798a703dc23538ebdb885c7ca7550646f5315f Mon Sep 17 00:00:00 2001
From: Marsell Kukuljevic <marsell@joyent.com>
Date: Tue, 26 Jun 2018 09:49:47 +0200
Subject: [PATCH] TRITON-548 Update javascriptlint and jsstyle in docker repo
 Reviewed by: Pedro P. Candel <pedro@joyent.com> Approved by: Pedro P. Candel
 <pedro@joyent.com>

---
 .gitmodules  | 4 ++--
 deps/jsstyle | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index 68b60c6..5f15143 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,9 +1,9 @@
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
-	url = https://github.com/davepacheco/javascriptlint.git
+	url = https://github.com/joyent/javascriptlint.git
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = https://github.com/davepacheco/jsstyle.git
+	url = https://github.com/joyent/jsstyle.git
 [submodule "deps/sdc-scripts"]
 	path = deps/sdc-scripts
 	url = https://github.com/joyent/sdc-scripts.git
diff --git a/deps/jsstyle b/deps/jsstyle
index d75b7ca..52dc973 160000
--- a/deps/jsstyle
+++ b/deps/jsstyle
@@ -1 +1 @@
-Subproject commit d75b7ca8308be17c80e2b120f2a01d4a0c20d8a8
+Subproject commit 52dc973cf64da11834eca7cf46ebce8518e3ee88
-- 
2.21.0

